acpi-prop.adoc

```diff
@@ -0,0 +1,25 @@
+[[acpi-props]]
+=== RVI-specific ACPI Device Properties
+
+This section defines the _DSD device properties cite:[DSD] in the `rscv-` namespace.
+
+Where explicit values are provided in a property definition, only these values must be used. System behavior with any other values is undefined.
+
+Request for additional property names in the `rscv-` namespace should be made as a git pull request to this document.
+
+[[acpi-props-uart]]
+==== Properties for UART Devices
+
+[width=100%]
+[%header, cols="10,5,25"]
+|===
+|  Property (`rscv-uart-*`) ^| Type | Description
+| `clock-frequency` | Integer | Clock feeding the IP block in Hz.
+3+| _A value of zero will preclude the ability to set the baud rate, or
+to configure a disabled device._
+| `reg-shift` | Integer | Quantity to shift the register offsets by.
+| `reg-io-width` | Integer | The size (in bytes) of the register accesses that should be performed on the device.
+3+| _1, 2, 4 or 8._
```

Comment:
Is the assumption that a write of a given length still only provides 1 byte of data?

Reply:
An io-width != 1 only makes sense if the reg-shift is not 1. Consider a UART with a reg-shift of 2, where it's single byte registers are aligned on 32-bit boundaries. There's been silicon that would only work (as in, the writes would only "write" something) if the accesses were also 32-bit. Such an access to a 32-bit aligned 8-bit registers would still provide only 1 byte of data, yes.

Reply:
Yep that's how NS16550 UARTs work.